packages/cli/generators/observer/templates/observer-template.ts.ejs

```diff
@@ -0,0 +1,33 @@
+import {bind} from '@loopback/context';
+import {
+  /* inject, Application, */
```

Comment:
Let's add `CoreBindings`, since the commented code below (`CoreBindings.APPLICATION_INSTANCE`) refers to it.